src/logic/events/AccessibilityEvents.swift

```diff
@@ -11,7 +11,8 @@ fileprivate func handleEvent(_ type: String, _ element: AXUIElement) throws {
     debugPrint("Accessibility event", type, type != kAXFocusedUIElementChangedNotification ? (try element.title() ?? "nil") : "nil")
     // events are handled concurrently, thus we check that the app is still running
     if let pid = try element.pid(),
-       try (!(pid == ProcessInfo.processInfo.processIdentifier && element.subrole() == kAXUnknownSubrole)) {
+       try ((pid != ProcessInfo.processInfo.processIdentifier && element.subrole() != kAXUnknownSubrole)
```

Comment:
In the ticket, people have reported that this change fixes the issue for them. While I'm *really* excited about these reports, I'm struggling to see how this change fixes it.

Previous code truth table:

Event is from AltTab | window subrole is unknown | event is focusedUIElement | process event?
-|-|-|-
true | true | true | no
true | true | false | no
true | false | true | yes
true | false | false | yes
false | true | true | yes
false | true | false | yes
false | false | true | yes
false | false | false | yes

New code truth table (changes in bold):

Event is from AltTab | window subrole is unknown | event is focusedUIElement | process event?
-|-|-|-
true | true | true | no
true | true | false | **yes**
true | false | true | **no**
true | false | false | yes
false | true | true | **no**
false | true | false | yes
false | false | true | yes
false | false | false | yes

If I didn't make a mistake in the tables, the three cases that changed are:

* Handling previously excluded events:
  * From AltTab, window subrole is unknown, event is not focusedUIElement
    * => It seems to me that we don't want to handle any of these events. They are from AT itself, from its main window. The **main window events should all be ignored**
* Not handling previously handled events:
  * event is focusedUIElement, from AltTab, window subrole is not unknown
    * => AT doesn't use tabs, so it's fine to ignore focusedUIElement events on its non-unknown windows. **I'm wondering if this one is not the key!** I'm thinking to test a build that only prevents this.
  * event is focusedUIElement, not from AltTab, window subrole is unknown
    * => I don't think we want to handle events from unknown subrole windows. I think this will **lead to issues** with some apps.

Could you please test a build with this condition? It changes only the case which I think is desirable, while not introducing the 2 cases which I think are not desirable. It may be that this change there is the key to fix the ticket. I can't test it on my machine, so could you? 🙇‍♂️

```swift
pid != ProcessInfo.processInfo.processIdentifier || element.subrole() == kAXUnknownSubrole || type != kAXFocusedUIElementChangedNotification
```

Reply:
Testing with this new conditional now.

Reply:
`voiceOverFocusedWindow()` gets called twice when the window first appears, which I think is because of `element.subrole() == kAXUnknownSubrole`. It doesn't seem to cause the same performance issues, and the method only gets called once per focus event afterwards.

Reply:
I'm sorry, I shared the wrong conditional. I got confused after making the tables I think. Here's the correct conditional that fixes the issue with `kAXFocusedUIElementChangedNotification` without introducing regressions, I believe:

```swift
pid != ProcessInfo.processInfo.processIdentifier || (element.subrole() != kAXUnknownSubrole && type != kAXFocusedUIElementChangedNotification)
```

I tested it yesterday and I triggered only one call to `voiceOverFocusedWindow` when summoning AT.

Reply:
I'll add a new build to the issue.